Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,12 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function addVat(price){
+    const vat = 0.2;
+    const priceWithoutPoundSign = Number(price.substring(1));
+    return  `£${(vat * priceWithoutPoundSign) + priceWithoutPoundSign}`;
+}
```

Comment:
The spec is a bit vague, but I think it will probably make more sense for this function to deal with a number directly and return a number.

But what's matter is that you know how to implement a function. So no change is necessary.